homeassistant/components/opentherm_gw/services.yaml

```diff
@@ -7,6 +7,22 @@ reset_gateway:
       description: The gateway_id of the OpenTherm Gateway.
       example: "opentherm_gateway"
 
+set_central_heating_ovrd:
+  description: >
+    Set the central heating override option on the gateway.
+    When overriding the control setpoint (via a set_control_setpoint service call with a value other than 0), the gateway automatically enables the central heating override to start heating.
+    This service can then be used to control the central heating override status.
+    To return control of the central heating to the thermostat, call the set_control_setpoint service with temperature value 0.
+    You will only need this if you are writing your own software thermostat.
+  fields:
+    gateway_id:
+      description: The gateway_id of the OpenTherm Gateway.
+      example: "opentherm_gateway"
+    ch_override:
+      description: >
+        The desired value for the central heating override. Use 0 to disable or 1 to enable.
+      example: "1"
```

Comment:
Should we make this `true`/`false` since its a binary state?

Reply:
Indeed, I will make the change. I noticed we do the same for `set_hot_water_ovrd`, I will address that in a separate PR.

Reply:
All done, should be resolved in 798cda1